src/index.html

```diff
@@ -0,0 +1,231 @@
+            </div>
+            <div class="program__card-title">Conclusion</div>
+            <div class="program__card-text">
+              We will conclude our conference here.
+            </div>
+          </div>
+        </div>
+        <a
+          class="section__link section__link--plain desktop-only"
+          href="/program"
+          >SEE THE WHOLE PROGRAM</a
+        >
+        <a class="section__link mobile-only" href="/program"
+          >Join JavaScript Conference</a
+        >
+      </div>
+    </section>
+    <section class="section">
+      <div class="container">
+        <h2 class="section__title">Featured Speakers</h2>
+        <div id="speakers" class="section__body section__body--desktop-grid">
+          <div class="speaker">
+            <img
+              class="speaker__image"
+              src="./assets/images/speaker-1.jpg"
+              alt="Speaker 1"
+            />
+            <div class="speaker__body">
+              <div class="speaker__name">Yochai Benkler</div>
+              <p class="speaker__occupation">
+                Berkman Professor of Entrepreneurial Legal Studies at Harvard
+                Law School
+              </p>
+              <p class="speaker__about">
+                Benkler studies commons-based peer production, and published his
+                senminal book The Wealth of Networks in 2006
+              </p>
+            </div>
+          </div>
+        </div>
+        <button class="section__more-btn" type="button">
+          <span>More</span>
+          <img src="./assets/icons/chevron-down.svg" alt="Chevron down icon" />
+        </button>
+      </div>
+    </section>
+    <section class="section section--dark desktop-only">
+      <div class="container">
+        <h2 class="section__title">Partners</h2>
+        <div class="section__body">
+          <img
+            class="partner"
+            src="./assets/images/partner-1.png"
+            alt="Partner 1"
+          />
+          <img
+            class="partner"
+            src="./assets/images/partner-2.png"
+            alt="Partner 2"
+          />
+          <img
+            class="partner"
+            src="./assets/images/partner-3.png"
+            alt="Partner 3"
+          />
+          <img
+            class="partner"
+            src="./assets/images/partner-4.png"
+            alt="Partner 4"
+          />
+          <img
+            class="partner"
+            src="./assets/images/partner-5.png"
+            alt="Partner 5"
+          />
+        </div>
+      </div>
+    </section>
+    <footer class="footer footer--light desktop-only">
+      <div class="footer__logo">
+        Event by
+        <img
+          height="40"
+          width="40"
+          src="./assets/images/git-nation.svg"
+          alt="Git Nation"
+        />
+        <strong>GitNation</strong>
+      </div>
+      <p class="footer__rights">2022 JS Nation, Some rights are reserved.</p>
+    </footer>
```

Comment:
You did great working on the changes asked by the previous reviewer on the footer but there is a small fix you can make to completely achieve the design given and as the previous reviewer requested.

- [ ] Kindly align the two sections in the footer to match the original design.

**The Original Design**
![image](https://user-images.githubusercontent.com/50941074/168941987-c32d6564-3cea-433d-b7de-4da5265ed54c.png)

**Your design**
![image](https://user-images.githubusercontent.com/50941074/168942113-45a57f5b-1efc-4506-8f5e-bccb09653f7c.png)

- [ ] It would be nice if you align the right section's text to the left so they are close to each other with just about a 50px distance. I believe that will do. 👍 Please fix that.
- [ ] Also please consider the same changes on the About page as well. 👍